src/Core/Notification.php

```diff
@@ -639,4 +639,48 @@ public function to(string $type, array $config = [])
 	{
 		return apply_filters(sprintf('notification/to/%s', $type), $this, $config);
 	}
+
+	/**
+	 * Serialized Notification instance.
+	 *
+	 * @return  array<string, mixed>
+	 */
+	public function __serialize(): array
+	{
+		/** @var array<string, mixed> */
```

Comment:
```suggestion
		/** @var NotificationData */
```

We have globally defined type, see phpstan.neon